src/markdown-pages/install-with-kurl/managing-nodes.md

```diff
@@ -0,0 +1,247 @@
+    ```
+
+* Verify that Ceph is in a healthy state by running one of the following `ceph status` commands in the `rook-ceph-tools` Pod in the `rook-ceph` namespace:
+
+    * **Rook Ceph v1.4.0 or later**:
+
+      ```
+      kubectl -n rook-ceph exec deploy/rook-ceph-tools -- ceph status
+      ```
+    * **Rook Ceph v1.0.0 or later**:
+
+      ```
+      kubectl -n rook-ceph exec deploy/rook-ceph-operator -- ceph status
+      ```
+
+      The output of the command shows `health: HEALTH_OK` if Ceph is in a healthy state.
+
+### (Recommended) Manually Rebalance Ceph and Remove a Node
+
+This procedure ensures that the data held in Rook Ceph is safely replicated to a new node before you remove a node. Rebalancing your data is critical for preventing against data loss that can occur when removing a node if the data stored in Ceph has not been properly replicated.
+
+To manually remove a node, you first use the ceph CLI to weigh the Ceph OSD to `0` on the node that you want to remove and wait for Ceph to rebalance the data across OSDs. Then, you can remove the OSD from the node, and finally remove the node.
+
+To manually rebalance data and remove a node:
+
+1. Review the [Rook Ceph and etcd Node Removal Requirements](#rook-ceph-and-etcd-node-removal-requirements) above.
+
+1. Complete the [Rook Ceph Cluster Prerequisites](#rook-ceph-cluster-prerequisites) above.
+
+1. Add the same number of new nodes to the cluster that you intended to remove. For example, if you intend to remove a total of two nodes, add two new nodes.
+
+   Ceph rebalances the existing placement groups to the new OSDs. To view progress, run:
+
+   ```
+   watch ceph status
+   ```
+
+   **Note**: The commands in this procedure assume that you created an interactive shell in the `rook-ceph-tools` or `rook-ceph-operator` Pod. For more information, see [Rook Ceph Cluster Prerequisites](#rook-ceph-cluster-prerequisites) above.
+
+1. After Ceph completes rebalancing, verify that Ceph is in a healthy state:
+
+   ```
+   ceph status
+   ```
+
+1. Reweight the OSD to `0` on the first node that you intend to remove:
+
+   ```
+   ceph osd reweight OSD_NAME 0
+   ```
+   Replace `OSD_NAME` with the name of the Ceph OSD on the node that you intend to remove.
+
+   Ceph rebalances the placement groups off the OSD that you specify in the `ceph osd reweight` command. To view progress, run `watch ceph status`.
+
+1. After the `ceph osd reweight` command completes, verify that Ceph is in a healthy state:
+
+   ```
+   ceph status
+   ```
+
+1. On the first node that you intend to remove, run the following ceph CLI command to mark the OSD as `down`:
+
+   ```
+   ceph osd down OSD_NAME
+   ```
+
+1. On the first node that you intend to remove, use the kubectl command-line tool to scale the OSD deployments to 0 replicas:
+
+   ```
+   kubectl scale deployment -n rook-ceph rook-ceph-osd-1 --replicas 0
+   ```   
+
+1. Remove the node.
```

Comment:
If we're going through all this trouble to scale things down etc, we really need to make people run the ok-to-remove command

Reply:
It looks like this
```
[rook@rook-ceph-tools-6fb84b545-js4rg /]$ ceph osd safe-to-destroy osd.1
OSD(s) 1 are safe to destroy without reducing data durability.
```

Reply:
Thanks, I'll add a step for this